docs/samples/browser-plugin-meetings/index.html

```diff
@@ -261,6 +292,8 @@ <h2 class="collapsible">Meeting Management</h2>
 
               <div>
                 <p class="note">NOTE: Choose your sending and receiving options for the meeting.</p>
+                <input id="meetings-join-multistream" type="checkbox" checked>
```

Comment:
this checkbox should probably be unticked by default on the sdk_v3 branch for now, so I'll change this before merging this PR

Reply:
done